docs/plugins/google-cloud.md

```diff
@@ -197,7 +197,7 @@ and logs.
 Provides an override that disables exporting traces while still exprting metrics
 and logs.
 
-#### disableLoggingIO
+#### disableLoggingInputAndOutput
```

Comment:
Should really just link to API ref for details like this:
https://genkit-js-api.web.app/interfaces/_genkit-ai_google-cloud.GcpTelemetryConfigOptions.html#disableloggingio
Maybe someday.

Reply:
W're going to overhaul these docs w/ David. We'll keep this in mind